pkg/suggestion/v1alpha2/katib_suggestion/rpc_service.py

```diff
@@ -0,0 +1,102 @@
+import grpc
+
+import api_pb2
+import api_pb2_grpc
+from .config import MANAGER_ADDRESS, MANAGER_PORT, SEARCH_ALGORITHM
+from .algorithm_register import ALGORITHM_REGISTER
+from .utils import get_logger
+from . import parsing_utils
+
+REQUIRES_PAST_OBSERVATIONS = ["bayesian_optimization", "grid_search"]
+
+
+class SuggestionService(api_pb2_grpc.SuggestionServicer):
+
+    def __init__(self, search_algorithm=SEARCH_ALGORITHM, logger=None):
+        self.manager_addr = MANAGER_ADDRESS
+        self.manager_port = MANAGER_PORT
+        self.logger = logger if (logger is not None) else get_logger()
+        self.search_algorithm = search_algorithm
```

Comment:
if search_algorithm is not in ALGORITHM_REGISTER, we should exit with right error message in __init__ instead of exception occurring in line 30 when GetSuggestions.
That is, the corresponding deployment should not be `Running` if search_algorithm is not supported